### PR TITLE
fix: Updated styles.yml File For ISSUE TEMPLATES

### PR DESCRIPTION
**Pull request by NarendraKoya999**
## Related Issue

- Fixes Issue No : #1924 

## Description

-  Added `styles.yml for GitHub ISSUE_TEMPLATES using for Raising Enhancement Requests.`

### CODE REVIEW

**Comment by NarendraKoya999** [comment]
Hi @priyankarpal . Please review on this PR as soon as possible and Add Valid Labels under GSSOC'23

File: .github/ISSUE_TEMPLATE/styles.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this template!
+  - type: textarea
+    id: style-idea
+    attributes:
+      label: What's the style idea?
+      placeholder: Add descriptions
+      value: 'We need to improve  '
```
Comment (priyankarpal) [inline .github/ISSUE_TEMPLATE/styles.yml:15]: remove 
```suggestion
     
```

File: .github/ISSUE_TEMPLATE/styles.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+name: 👯‍♂️ Style Changing Request
+description: Suggest a style designs
+title: '[style]: '
+labels: ['enhancement']
```
Comment (priyankarpal) [inline .github/ISSUE_TEMPLATE/styles.yml:4]: add these labels
```suggestion
labels: ["✨ goal: improvement","🚦status: awaiting triage","🕹 aspect: interface"]
```

File: .github/ISSUE_TEMPLATE/styles.yml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+      label: What's the style idea?
+      placeholder: Add descriptions
+      value: 'We need to improve  '
+    validations:
+      required: true
+  - type: textarea
+    id: screenshots
+    attributes:
+      label: Add screenshots
+      description: Add screenshots to see the demo
```
Comment (priyankarpal) [inline .github/ISSUE_TEMPLATE/styles.yml:22]: remove
```suggestion
      
```

File: .github/ISSUE_TEMPLATE/styles.yml [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+      placeholder: Add screenshots
+      value: 'Add screenshots'
```
Comment (priyankarpal) [inline .github/ISSUE_TEMPLATE/styles.yml:24]:
```suggestion
      
```

File: .github/ISSUE_TEMPLATE/styles.yml [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+  - type: checkboxes
+    id: terms
+    attributes:
+      label: Code of Conduct
```
Comment (priyankarpal) [inline .github/ISSUE_TEMPLATE/styles.yml:28]: change to `Record`
```suggestion
      label: "Record"
```

File: .github/ISSUE_TEMPLATE/styles.yml [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+      description: By submitting this issue, you agree to follow our Code of Conduct
```
Comment (priyankarpal) [inline .github/ISSUE_TEMPLATE/styles.yml:29]:
```suggestion
      
```

File: .github/ISSUE_TEMPLATE/styles.yml [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+      options:
+        - label: I agree to follow this project's Code of Conduct
```
Comment (priyankarpal) [inline .github/ISSUE_TEMPLATE/styles.yml:31]:
```suggestion
        - label: I have checked the existing [issues](https://github.com/priyankarpal/ProjectsHut/issues)
          required: true
```

File: .github/ISSUE_TEMPLATE/styles.yml
Comment (priyankarpal) [inline .github/ISSUE_TEMPLATE/styles.yml:1]: At the end add these code sample also, @NarendraKoya999 

```

        - label: I have read the [Contributing Guidelines](https://github.com/priyankarpal/ProjectsHut/blob/main/contributing.md)
          required: true

        - label: I agree to follow this project's [Code of Conduct](https://github.com/priyankarpal/ProjectsHut/blob/main/CODE_OF_CONDUCT.md)
          required: true

        - label: I'm a GSSoC'23 contributor

        - label: I want to work on this issue
       
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/styles.yml
Comment (priyankarpal) [reply .github/ISSUE_TEMPLATE/styles.yml:1]: @NarendraKoya999 please add the code sample

Comment (NarendraKoya999) [reply .github/ISSUE_TEMPLATE/styles.yml:1]: Added above Lines. Please Check @priyankarpal

**Review by priyankarpal — APPROVED** [review]

**Comment by priyankarpal** [comment]
Hey @NarendraKoya999  ! 👋

For secure collaboration, remember to check out GitHub's commit signature verification feature. It helps ensure the authenticity of commits and prevents tampering.

Learn how it works: [GitHub Docs](https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification)

Happy coding! 😄

**Pull request merged**